### PR TITLE
Adding cluster scaling

### PR DESCRIPTION
**Pull request by dry923**
Depends-On: 401

Adding a cluster scaling function/test/workload. It takes a desired scale size and evenly spreads it over the worker machine sets and waits for the scale up/down to complete.

### CODE REVIEW

File: snafu/scale_wrapper/trigger_scale.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+logger = logging.getLogger("snafu")
+
+
+class Trigger_scale():
+    def __init__(self, args):
+        self.uuid = args.uuid
+        self.user = args.user
+        self.scale = args.scale
+        self.cluster_name = args.cluster_name
+        if args.results :
```
Comment (jtaleric) [inline snafu/scale_wrapper/trigger_scale.py:34]: how would init come with the results?

Comment (dry923) [reply snafu/scale_wrapper/trigger_scale.py:34]: It's a carryover from the workload I copied the base skel from. I'll clean it up

**Review by rsevilla87 — APPROVED** [review]
Apart from the renaming suggestion made by @jtaleric. LGTM!. Nice job @dry923

**Comment by dry923** [comment]
> consider changing the name of the module to `scale_openshift_wrapper`, thoughts?

naming updated. good suggestion

**Comment by rsevilla87** [comment]
Hey @dry923 mind you fix the linting errors?

**Review by jtaleric — APPROVED** [review]
LGTM

**Review by rsevilla87 — APPROVED** [review]
LGTM!

**Pull request merged**